### ISSUE TITLE
No sound output?

### ISSUE THREAD

**Issue by haydendavenport** [etscrivner/sdl_audio_circular_buffer#1]
Hi,

Thanks for making this and the tutorial. It was one of the only things I could find related to audio and SDL without using the SDL_mixer library.

Unfortunately, I wasn't able to get this code to output sound. At first I tried writing it based on the tutorial, but got lost at this line: `SampleIntoAudioBuffer(AudioBuffer, &SampleSineWave);` because there wasn't a globally defined AudioBuffer as far as I could tell. So I just copy/pasted this code in a fresh project and compiled it. But all it did was open a window with no idea. 

Any idea what might be going wrong?

Thanks again, I really appreciate the effort you put into sharing this!

**Comment by etscrivner** [comment]
@haydendavenport So you can find the definition of `AudioBuffer` in the provided source code here: https://github.com/etscrivner/sdl_audio_circular_buffer/blob/master/src/circular_buffer.cpp#L247

Just did a fresh build on a new machine and the audio was working for me.

What OS are you building on? Are you attempting to build this repository?

**Comment by haydendavenport** [comment]
Hey Eric,

Thanks for the additional info! I tried building from source twice using Casey Muratori's "unity build" style. Everything compiled fine other than needing to specify args/argv in main for some reason. But I'm on Windows 10, and I just saw that it hasn't been tested on Windows, so maybe that's the issue? Although that seems strange since it's SDL after all... Weird... Fortunately though, I did get something going between looking at this code,the handmade hero code, and trying different stuff. Thanks a bunch for taking the time to write the article and provide your code. It helped me a ton!
